### PR TITLE
[LT] added lithuanian language sentences

### PR DESCRIPTION
**Pull request by vilius365**
Issues - "Lithuanian sentences that should be supported #54"

### CODE REVIEW

**Comment by synesthesiam** [comment]
@vilius365 Are you able to sign the CLA?

**Comment by vilius365** [comment]
I believe its done now. Can you confirm. Also as far as I understand I need to add the tests right?

**Comment by balloob** [comment]
Please sign the CLA for us to proceed. It needs to be signed by the same GitHub user that made the commits.

**Comment by vilius365** [comment]
Added relevant email address to my github account

**Comment by vilius365** [comment]
closed the issue by accident, also recommited the code cla should be ok

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (balloob) [inline .vs/VSWorkspaceState.json:1]: please do not include this file.

**Comment by balloob** [comment]
Hey @vilius365, your tests were failing because the branch was too far behind. I've updated your branch and updated `languages.yaml`

**Pull request closed without merging**